yarn.lock

```diff
@@ -2256,7 +2256,7 @@
     jsonwebtoken "^8.5.1"
     lodash "^4.17.11"
 
-"@tryghost/members-stripe-service@0.8.4", "@tryghost/members-stripe-service@^0.8.4":
+"@tryghost/members-stripe-service@0.8.4":
```

Comment:
I think `@tryghost/members-stripe-service` needs bumping in Ghost as well, otherwise afaict we have two different versions of it 🤔 
cc @allouis is this expected?

Reply:
This is fine not to be bumped - yeah it's expected, I'll fix it though, it's because we still have it as a dependency of members-api